README.md

```diff
@@ -1,3 +1,6 @@
+[![NPM Version](https://img.shields.io/npm/v/bzr/bazaar-js.svg?style=flat)](https://www.npmjs.com/package/@bzr/bazaar)
```

Comment:
This NPM badge

Reply:
Maybe it requires a `latest` tag or something. Will figure it out later... its not super important.